src/rand/null.jl

```diff
@@ -43,3 +43,22 @@ species.
 function null2(N::BinaryNetwork)
   return linearfilter(N; α=[0.0, 0.5, 0.5, 0.0])
 end
+
+
+"""
+    null2mult(N::BinaryNetwork)
+
+Given a matrix `A`, `null2mult(A)` returns a matrix with the same dimensions, where
+every interaction happens with a probability equal to the product of the degree
+of each species.
+"""
+function null2mult(N::BinaryNetwork)
```

Comment:
I would give it a new name -- it's really different from `null2`, which is doing averages. What about `null4`? At this point, we might as well call this Type IV null model.

Reply:
done